### ISSUE TITLE
Atypical hemolytic-uremic syndrome 

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6871]
**Preferred term label:**
Atypical hemolytic-uremic syndrome 

**Synonyms**
aHUS, familial HUS, genetic HUS

**Definition**
Genetic or acquired defect in regulation of complement activation leading to complement overactivation.
PMID: 19846853, PMID: 28416508

**Parent term (use hpo.jax.org/app)**
Hemolytic-uremic syndrome HP:0005575

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Associated Genes**
CFH, CFH/CFHRs, CFHR1/CFHR3, CFHR1/CFHR4, MCP, CFI, C3, CFB, THBD, DGKE 

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
see also @RichardKlaus one issue is that the HPO does not code etiologies. I think the specific kinds of HUS should be encoded as diseases in Mondo (e.g., https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0034103).
